adam/curriculum_to_html.py

```diff
@@ -127,6 +127,43 @@ def _dump_instance_group(
         with open(output_destination, "w") as html_out:
             html_out.write(f"<head>\n\t<style>{CSS}\n\t</style>\n</head>")
             html_out.write(f"\n<body>\n\t<h1>{title} - {instance_group.name()}</h1>")
+            html_out.write("\n<h2>How to read</h2>")
+            html_out.write(
+                "\n<p>A Situation is curriculum-designer-facing; it is not accessible to the learner. "
+                "Names on Perceptions are handles for debugging; they are not accessible to the "
```

Comment:
@elizlee, from my discussion with Marjorie we might want to add a sentence here describing how to do the mapping from the Situation Objects to the perceived objects. (Example: Baby becomes `person_0` which might not be intuitive to follow)